pom.xml

```diff
@@ -67,11 +67,6 @@
     <tag>HEAD</tag>
   </scm>
 
-  <properties>
-    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
it's intended: otherwise it complains on some jenkins.getActiveInstance coming from credentials plugin 2.1.5 which does not exists in 1.580. I need the latest on credentials plugin